pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -557,5 +561,41 @@ func (a *Bootstrap) Load(f asset.FileFetcher) (found bool, err error) {
 	}
 
 	a.File, a.Config = file, config
+	warnIfCertificatesExpired(a.Config)
 	return true, nil
 }
+
+// warnIfCertificatesExpired checks for expired certificates and warns if so
+func warnIfCertificatesExpired(config *igntypes.Config) {
+	expiredCerts := 0
+	for _, file := range config.Storage.Files {
+		if strings.HasSuffix(file.Path, ".crt") && file.Contents.Source != nil {
+			// ^data:text/plain;charset=utf-8;base64,LS0.+==
+			i := strings.Index(*file.Contents.Source, ",LS0")
+			if i == -1 {
+				continue
+			}
```

Comment:
https://github.com/vincent-petithory/dataurl I think ignition uses this..

Reply:
Fixed.